Add file Markdown

ArtyomSenokosov opened the pull request.

ArtyomSenokosov committed:
Add file Markdown

sonianicoletti reviewed (commented):
One thing needs to be changed. The chunking of the markdown documents happens only at the Level 1 headers (only one "#") and the chunks shouldn't be more than 1000 words long. Please split the longer chunks into 2 or 3 chunks (yes, even if it makes more sense for it to be a Level 2 header you should change it to Level 1).
The rest is good!

ArtyomSenokosov committed:
Update Essence_Kernel_Automation.md

1. All sectioning occurs by level 1 headings (#).
2. Each section contains less than 1000 words (the longest section is "2. Related Work", and it contains 594 words).

sonianicoletti reviewed (approved):
OK
